src/main/java/org/opensearch/security/auditlog/sink/WebhookSink.java

```diff
@@ -328,7 +329,7 @@ public KeyStore run() {
 
 					} else {
 					    return PemKeyReader.loadKeyStore(PemKeyReader.resolve(SSLConfigConstants.SECURITY_SSL_TRANSPORT_TRUSTSTORE_FILEPATH, settings, configPath, false)
-			                    , settings.get(SSLConfigConstants.SECURITY_SSL_TRANSPORT_TRUSTSTORE_PASSWORD, SSLConfigConstants.DEFAULT_STORE_PASSWORD)
+			                    , SecureSSLSettings.SSLSetting.SECURITY_SSL_TRANSPORT_TRUSTSTORE_PASSWORD.getSetting(settings)
```

Comment:
Nit: My preference is to use static imports for enum references like this and in several other files